packages/edit-site/src/components/page-pages/index.js

```diff
@@ -117,6 +120,15 @@ export default function PagePages() {
 										postType: page.type,
 										canvas: 'edit',
 									} }
+									onClick={ ( event ) => {
```

Comment:
This shouldn't be in the "fields" config IMO and also "fields" config shouldn't depend on local state here, it should only use things passed as props. This of this "render" function as the "edit" function of blocks. It might be defined in a very different file/package...

Reply:
Fixed 😊